include/libdnf-cli/argument_parser.hpp

```diff
@@ -168,6 +168,12 @@ class ArgumentParser {
         /// Can contain this argument. Groups of conflicting argument can be used.
         void set_conflict_arguments(std::vector<Argument *> * args) noexcept { conflict_args = args; }
 
+        /// Adds a conflicting argument. Also adds reverse conflict.
+        void add_conflict_argument(Argument & conflict_arg);
+
+        /// Add conflict arguments from another argument. Also adds reverse conflicts.
+        void add_conflict_arguments_from_another(Argument & src_arg);
```

Comment:
I also originally wanted to name the method `copy_conflict_arguments_from_another`. But then I wondered if it would be confusing. If conflicting arguments already exist (eg added explicitly with `add_conflict_argument`) using `copy_conflict_arguments_from_another` will add more conflicts and leave the existing ones. That's why I called `add`, that it will not replace existing conflicts, but add new ones. But maybe I'm overthinking it.